site/src/api/queries/users.ts

```diff
@@ -83,3 +88,76 @@ export const authMethods = () => {
     queryFn: API.getAuthMethods,
   };
 };
+
+export const me = () => {
+  return {
+    queryKey: ["me"],
+    queryFn: async () =>
+      getMetadataAsJSON<User>("user") ?? API.getAuthenticatedUser(),
+  };
+};
+
+export const hasFirstUser = () => {
+  return {
+    queryKey: ["hasFirstUser"],
+    queryFn: API.hasFirstUser,
```

Comment:
I think this is okay for now since it is using MSW on the related tests... this is something we should talk about and get some "consensus" about what to use to mock API requests.